crates/issue/src/benchmarking.rs

```diff
@@ -68,7 +71,7 @@ fn mine_blocks<T: crate::Config>() {
         .build();
 
     let mut prev_hash = block.header.hash;
-    for _ in 0..100 {
+    for _ in 0..600 {
```

Comment:
any particular reason for this? If so, would be good to add a comment to explain the reason